### PR TITLE
Updated the Language selector icon

### LINKED ISSUE #3372: Change language-picker icon so it's not the Google Translate icon
I saw that a language picker was added in #2294, that's great!

However, I noticed that the language picker icon uses the Google Translate icon:
Google Translate icon

I think this makes it look like the site will be automatically translated with Google Translate, rather than actual human translations.

### PR DESCRIPTION
**Pull request by DantrazTrev**
Fixes #3372 

I've changed the language selector icon to a better one, that is easily identifiable for all users 
image

As mentioned in the issue
The current icons makes it look like the site will be automatically translated with Google Translate, rather than actual human translations.

### CODE REVIEW

**Comment by XhmikosR** [comment]
@nschonni @Trott thoughts?

File: layouts/partials/header.hbs
Comment (XhmikosR) [inline layouts/partials/header.hbs:27]: We should try and keep the previous dimensions

**Comment by Trott** [comment]
> @nschonni @Trott thoughts?

I don't have an opinion on this one and am happy to defer to you, @nschonni, and anyone else in the website group who might be paying more attention to this particular interface element.

**Comment by XhmikosR** [comment]
I'm not sure I like this new icon TBH. But the previous one is wrong too, because it's from Google translate.

**Comment by nschonni** [comment]
Where did the new icon come from? I'm not sure if the current one was properly sourced either 😄

**Comment by XhmikosR** [comment]
That I don't know either. Better put this on hold then?

**Comment by ghost** [comment]
What about this article about the language icon? Use for free.

http://www.languageicon.org/

**Comment by DerekNonGeneric** [comment]
> What about this article about the language icon? Use for free.
> 
> http://www.languageicon.org/

It looks like we are going to be going w/ that suggestion. Thanks for the link, I was able to convert that to an optimized SVG for whoever wants to do the honors. @DantrazTrev, you may be able to simply modify your PR to use the following SVG.

https://codepen.io/DerekNonGeneric/pen/bGroexG

Also, please take dark mode into account if you do end up going for it as this icon would need to be made into a lighter color to contrast the dark background.

**Review by DerekNonGeneric — CHANGES_REQUESTED** [review]
> What about this article about the language icon? Use for free.
> 
> http://www.languageicon.org/

It looks like we are going to be going w/ that suggestion. Thanks for the link, I was able to convert that to an optimized SVG for whoever wants to do the honors. @DantrazTrev, you may be able to simply modify your PR to use the following SVG.

https://codepen.io/DerekNonGeneric/pen/bGroexG

Also, please take dark mode into account if you do end up going for it as this icon would need to be made into a lighter color to contrast the dark background.

**Pull request closed without merging**
